src/Ombi.Notifications/Agents/DiscordNotification.cs

```diff
@@ -200,25 +242,55 @@ protected override bool ValidateConfiguration(DiscordNotificationSettings settin
             {
                 var discordBody = new DiscordWebhookBody
                 {
-                    content = model.Message,
+                    // content = model.Message,
```

Comment:
Just realised this will break all the other notifications I haven't implemented yet. Working on a fix.